__tests__/components/cards/wallet/WalletHint.test.tsx

```diff
@@ -0,0 +1,37 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { Wallet } from "@/types/wallet";
+import WalletHint from "@/components/cards/wallet/WalletHint";
+import { mockWallet } from "@__mocks__/fixtures/wallet";
+
+const wallet: Wallet = {
+  ...mockWallet,
+  payouts: [
+    { amount: 100, token: "ETH" },
+    { amount: 200, token: "BTC" },
+  ],
+};
+
+describe("WalletHint component", () => {
+  it("should render WalletHint component with the correct number of Hint components", () => {
+    render(<WalletHint wallet={wallet} />);
+    const textElements = screen.getAllByText("profile.wallet.payout.text");
+    expect(textElements).toHaveLength(wallet.payouts.length);
+  });
+
+  it("renders the correct number of Hint components", () => {
+    render(<WalletHint wallet={wallet} />);
+    const hints = screen.getAllByText("profile.wallet.payout.text");
+    expect(hints).toHaveLength(wallet.payouts.length);
```

Comment:
It good already that you are checking the length; it might also be better if you can check the amount and the token is well rendered in the component